server/utils/cache.js

```diff
@@ -0,0 +1,32 @@
+const NodeCache = require('node-cache');
+const cache = new NodeCache({ stdTTL: 1 * 60 });
+
+parseUrl = req => {
```

Comment:
Can we make these `const`s instead of globals?

Reply:
next commit reflects changing these to const, good catch!